invenio_jobs/assets/semantic-ui/js/invenio_jobs/administration/RunButton.js

```diff
@@ -0,0 +1,116 @@
+// This file is part of InvenioRDM
+// Copyright (C) 2024 CERN
+//
+// Invenio RDM Records is free software; you can redistribute it and/or modify it
+// under the terms of the MIT License; see LICENSE file for more details.
+
+import { i18next } from "@translations/invenio_app_rdm/i18next";
+import PropTypes from "prop-types";
+import React, { Component } from "react";
+import { http } from "react-invenio-forms";
+import {
+  Button,
+  Dropdown,
+  DropdownMenu,
+  Form,
+  FormField,
+  FormInput,
+  TextArea,
+} from "semantic-ui-react";
+
+export class RunButton extends Component {
+  constructor(props) {
+    super();
+
+    this.state = {
+      title: "Manual run",
+      config: JSON.stringify(props.config, null, "\t"),
+      queue: "low",
+      loading: false,
+    };
+  }
+
+  handleChange = (e, { name, value }) => this.setState({ [name]: value });
+
+  handleSubmit = async () => {
+    this.setState({ loading: true });
+
+    const { jobId, onError } = this.props;
+    const { title, config, queue } = this.state;
+
+    const userConfigJSON = JSON.parse(config); // catch JSON errors?
+
+    const runData = {
+      title: title,
+      args: userConfigJSON,
+      queue: queue,
+    };
+
+    try {
+      await http.post("/api/jobs/" + jobId + "/runs", runData);
+      this.setState({ loading: false });
+    } catch (error) {
+      this.setState({ loading: false });
+      onError(error);
+    }
+  };
```

Comment:
Move status setter to the bottom

```suggestion
    try {
      await http.post("/api/jobs/" + jobId + "/runs", runData);
    } catch (error) {
      onError(error);
    }
    this.setState({ loading: false });
  };
```